docs/api/enums.rst

```diff
@@ -1867,3 +1867,90 @@ of :class:`enum.Enum`.
     .. attribute:: guild_only
 
         Represents a scheduled event that is only available to members inside the guild.
+
+.. class:: AutoModTriggerType
```

Comment:
Maybe reference this class in the trigger metadata's docs as a place to find what attrs may be available, or add a table similar to [`Member.edit`](https://docs.pycord.dev/en/stable/api/models.html#discord.Member.edit)

Reply:
The docs for `AutoModTriggerMetadata` already includes the applicable trigger types on each attribute, the general format for which I copied from somewhere else and have since forgotten.

```
keyword_filter: List[:class:`str`]
    A list of substrings to filter.
    Only for triggers of type :attr:`AutoModTriggerType.keyword`.
```

I do like the table format, which is used in the Discord docs, but I also like it as-is.